prymer/primer3/primer3_weights.py

```diff
@@ -80,3 +83,34 @@ def to_input_tags(self) -> dict[Primer3InputTag, Any]:
             Primer3InputTag.PRIMER_WT_TM_GT: self.primer_tm_gt,
         }
         return mapped_dict
+
+
+@dataclass(frozen=True, init=True, slots=True)
+class ProbeWeights:
+    """Holds the weights that Primer3 uses to adjust penalties
+    that originate from the designed internal probe(s)."""
```

Comment:
**doc** Make the summary line one line, and add an `Attributes` block


https://google.github.io/styleguide/pyguide.html#384-classes